plenum/common/signer_did.py

```diff
@@ -16,11 +16,25 @@ class DidIdentity:
     abbr_prfx = '~'
 
     def __init__(self, identifier, verkey=None, rawVerkey=None):
+        if verkey is None and rawVerkey is None:
+            if identifier:
+                self._identifier = identifier
+                self._verkey = None
+                return
```

Comment:
set `self.abbreviated` to None just after `__init__` starts  so that your class will always have the attribute, None signifies the value is not applicable